app/collections/collaborators.cjsx

```diff
@@ -150,11 +150,11 @@ RoleRow = React.createClass
       @props.roleSet.delete()
 
     promise
-      .catch (error) =>
-        @setState { error }
       .then =>
         callback =>
-          @setState saving: false
+        @setState saving: false
```

Comment:
Ah right. No, I was asking because I'm not familiar with this bit of code and wasn't sure why it had changed.